lua/esqueleto/utils.lua

```diff
@@ -158,32 +163,35 @@ M.selecttemplate = function(templates, opts)
   local templatenames = vim.tbl_keys(templates)
   table.sort(templatenames, function(a, b) return a:lower() < b:lower() end)
 
+  local uv = vim.loop or vim.uv
   -- If only one template, write and return early
   if #templatenames == 1 and opts.autouse then
-    M.writetemplate(vim.loop.fs_realpath(templates[templatenames[1]]), opts)
+    M.writetemplate(uv.fs_realpath(templates[templatenames[1]])--[[@as string]], opts)
     return nil
   end
 
   -- Select template
-  vim.ui.select(templatenames, { prompt = "Select skeleton to use:" }, function(choice)
+
+  -- wrap to coroutine to have possibility to async function
+  ---@see esqueleto.selectors.builtin
+  coroutine.wrap(function()
```

Comment:
I think only one breaking chang at that place, if it work correct, then it must work correct either errors exists at another functions

Reply:
Using this PR, I get the following error:

~~~
Error detected while processing BufNewFile Autocommands for "*":
Error executing lua callback: .../latest/nvim-macos-arm64/share/nvim/runtime/filetype.lua:36: BufNewFile Autocommands for "*"..FileType Autocommands for "*"..FileType Autocommands for "*"..FileType Autocomman
ds for "*"..FileType Autocommands for "*"..FileType Autocommands for "*"..FileType Autocommands for "*"..FileType Autocommands for "*"..FileType Autocommands for "*"..FileType Autocommands for "*": Vim(append
):Error executing lua callback: ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:182: ...queleto_dev/lazy/plenary.nvim/lua/plenary/popup/init.lua:122: Vim:E218: Autocommand nesting too deep
stack traceback:
        [builtin#36]: at 0x01008f59f0
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:182: in function 'selecttemplate'
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:208: in function 'inserttemplate'
        ...carlosvigil/git/esqueleto.nvim/lua/esqueleto/autocmd.lua:19: in function <...carlosvigil/git/esqueleto.nvim/lua/esqueleto/autocmd.lua:15>
        [builtin#36]: at 0x01008f59f0
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:182: in function 'selecttemplate'
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:208: in function 'inserttemplate'
        ...carlosvigil/git/esqueleto.nvim/lua/esqueleto/autocmd.lua:19: in function <...carlosvigil/git/esqueleto.nvim/lua/esqueleto/autocmd.lua:15>
        [builtin#36]: at 0x01008f59f0
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:182: in function 'selecttemplate'
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:208: in function 'inserttemplate'
        ...
        [builtin#36]: at 0x01008f59f0
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:182: in function 'selecttemplate'
        ...s/carlosvigil/git/esqueleto.nvim/lua/esqueleto/utils.lua:208: in function 'inserttemplate'
        ...carlosvigil/git/esqueleto.nvim/lua/esqueleto/autocmd.lua:19: in function <...carlosvigil/git/esqueleto.nvim/lua/esqueleto/autocmd.lua:15>
        [C]: in function 'nvim_cmd'
        .../latest/nvim-macos-arm64/share/nvim/runtime/filetype.lua:36: in function <.../latest/nvim-macos-arm64/share/nvim/runtime/filetype.lua:35>
        [C]: in function 'pcall'
        vim/shared.lua: in function <vim/shared.lua:0>
        [C]: in function '_with'
        .../latest/nvim-macos-arm64/share/nvim/runtime/filetype.lua:35: in function <.../latest/nvim-macos-arm64/share/nvim/runtime/filetype.lua:10>
stack traceback:
        [C]: in function '_with'
        .../latest/nvim-macos-arm64/share/nvim/runtime/filetype.lua:35: in function <.../latest/nvim-macos-arm64/share/nvim/runtime/filetype.lua:10>
~~~